org.alloytools.alloy.application/src/main/java/templates/TestTemplateFile.vm

```diff
@@ -375,7 +421,6 @@ $padding                ${trans.getGuardConditions().get($lastIndex)}):
 $padding            return False
                 #end
             #end
-$padding        logging.log(GCLOG,f"Guard of [${concState.getName()}_${trans.getTransName()}] triggered.")
```

Comment:
Just curious, why are we removing the logging statements?

Reply:
When a transition is taken a message is printed to the user, so the logging statements are redundant.